daylay92/ListIt
# Integrate create and fetch functionality

#### What does this PR do?
Adds integration of client and server functionalities to enable users to view and create bucket lists.
#### Description of Task to be completed?
- Implement route guard to prevent unauthorized access to protected routes.
- Implement action creators to facilitate interactions with the redux store.
- Implement corresponding the connection of components to the store through the connect function of react-redux.
#### How should this be manually tested?
- Clone the repo and cd into the project folder
- Checkout to `ft-integrate-create-fetch` branch and run `npm install`.
- Also run `npm run client-install`.
- Manually create an Account or signin
- Create and view your created bucket lists.
- Please see the README.md file for a guide on how to locally install the App
